### PR TITLE
Including playbook_rabbitmq_perms.yml

### PR DESCRIPTION
**Pull request by bayrem-k**
Including playbook_rabbitmq_perms.yml

### CODE REVIEW

**Comment by ppetit** [comment]
Bayrem can you please a license header in both files.
I would rather not use capital letters in an variable names.

**Comment by ppetit** [comment]
Also pleas remove the lines that are commented.

**Comment by bayrem-k** [comment]
Done Patrick

**Comment by ppetit** [comment]
Code change proposal merged. Thanks!

File: playbook_rabbitmq_perms.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  serial: 1
+  gather_facts: "{{ osa_gather_facts | default(True) }}"
+  user: root
+  tasks:
+  - name: extract list of rabbitmq vhosts
+    shell: "rabbitmqctl list_vhosts"
+    register: rabbitmq_vhosts
+
+  - name: attribute permissions for monitoring user on rabbitmq vhosts
+    shell: 'rabbitmqctl set_permissions -p {{ item }} monitoring "^$" ".*" "^$" '
```
Comment (obourdon) [inline playbook_rabbitmq_perms.yml]: NIT remove trailing space in shell command

Comment (obourdon) [reply playbook_rabbitmq_perms.yml]: Copyright missing

**Pull request merged**